infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -63,14 +63,14 @@
         <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
             <fileNamePattern>${LOG_DIRECTORY}/tx-log.%i.log.gz</fileNamePattern>
             <minIndex>1</minIndex>
-            <maxIndex>75</maxIndex>
+            <maxIndex>100</maxIndex>
         </rollingPolicy>
         <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
-            <maxFileSize>200MB</maxFileSize>
+            <maxFileSize>100MB</maxFileSize>
         </triggeringPolicy>
         <encoder>
             <pattern>
-                %date{yyyy-MM-dd'T'HH:mm:ss.SSSXXX, UTC} | %30.30thread | %msg%n%xException
+                %date{yyyy-MM-dd'T'HH:mm:ss.SSSXXX, UTC} | %11.11thread | %msg%n%xException
```

Comment:
Isn't it possible to miss out some thread name information since it's only 11 characters?

Something like orchestrator-0/1/2, we might want more than 11 characters here

Reply:
Good point. I find some longer thread names than others. You can adjust the width accordingly.
```
root@manager1:/var/log/corfu# less corfu.9000.log  | awk '{print $5}' | sort | uniq

AutoCommitService
Cmpt-9000-chkpter
CorfuRuntime-0
DetectionWorker-0
DetectionWorker-1
DetectionWorker-2
LogUnit-0
LogUnit-1
LogUnit-2
LogUnit-3
LogUnit-BatchProcessor-0
LogUnit-Maintenance-0
ViewsGarbageCollector
client-0
```

Reply:
This file - `tx-log.log` (for which the pattern is meant for) contains only handleWrite log lines by `LogUnit-n` thread. For example -

```
2024-01-10T09:33:26.228Z |   LogUnit-2 | 0, 2825054, DATA: [36aaac7f=2825053, 8064fc71=2825051]
2024-01-10T09:33:26.276Z |   LogUnit-3 | 0, 2825055, DATA: [36aaac7f=2825054, f5c02263=2825050]
2024-01-10T09:33:26.336Z |   LogUnit-0 | 0, 2825056, DATA: [36aaac7f=2825055, 8064fc71=2825054]
2024-01-10T09:33:26.387Z |   LogUnit-1 | 0, 2825057, DATA: [36aaac7f=2825056, f5c02263=2825055]
2024-01-10T09:33:26.550Z |   LogUnit-2 | 0, 2825058, DATA: [36aaac7f=2825057, 8064fc71=2825056]
```

Hence 11 is sufficient.